### ISSUE TITLE
Convert TensorFlow Tensor into numpy before assigning values to neural net

### ISSUE THREAD

**Issue by eemailme** [eemailme/feedly#32]
Convert TensorFlow Tensor into numpy before assigning values to neural net<br><br>
https://stackoverflow.com/questions/46935897/convert-tensorflow-tensor-into-numpy-before-assigning-values-to-neural-net<br><br><p>I am working with the <a rel="nofollow noreferrer" href="https://github.com/carpedm20/DCGAN-tensorflow">DCGAN code</a>. I need to modify <a rel="nofollow noreferrer" href="https://github.com/carpedm20/DCGAN-tensorflow/blob/master/model.py#L127">the reward</a> that is given to one of the neural nets by adding a function that would take the output of this neural net, analyse it, and issue a penalty on it. So my loss function would look like:</p>
<pre>
<code>self.g_loss = self.g_loss + self.penalty
</code>
</pre>
<p>Problem is</p>
<ul>
<li>
<p>this penalty function only takes the numpy arrays as an input (I have no way of modifying this),</p>
</li>
<li>
<p>neural network output is a tf.tensor,</p>
</li>
<li>
<p>and as the values haven't been assigned to the neural net yet (technically it hasn't been built yet) I can't run neither <code>.eval()</code> nor <code>sess.run()</code>.</p>
</li>
</ul><p>So how would I convert a tensorflow tensor into numpy array in this case?</p>
<br><br><br><br>
Everything<br><br>
via Newest questions tagged python - Stack Overflow https://stackoverflow.com/questions/tagged/?tagnames=python&amp;sort=newest<br><br>
October 25, 2017 at 06:30AM
